src/components/Orders/OldOrders.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import OldOrder from './OldOrder';
+
+function OldOrders(props) {
+    return (
+        <div className="menu-wrapper">
+            <Link to="/menu"><h5 className="menu__back-to-menu">Back to Menu</h5></Link>
+            <h1 className="menu__heading">Order History</h1>
+            <div className={"old-orders__empty " + (Object.keys(props.oldOrders).length > 0 ? "hidden" : "")}>
```

Comment:
The condition could be written a little shorter using `Object.keys(props.oldOrders).length` since it will evaluate to `true` when length is greater than 0